pkgs/top-level/all-packages.nix

```diff
@@ -7144,7 +7144,7 @@ with pkgs;
 
   grabserial = callPackage ../development/tools/grabserial { };
 
-  mypy = callPackage ../development/tools/mypy { };
+  mypy = python3Packages.callPackage ../development/tools/mypy { };
```

Comment:
This seems to be incorrect. `mypy` is an application, is therefore not in `python-packages.nix` or `python-modules` and thus should only be called directly from `all-packages.nix`.